quirks/websites-with-shared-credential-backends.json

```diff
@@ -218,4 +218,12 @@
         "lookmark.io",
         "lookmark.link"
     ]
+    [
```

Comment:
Some commas are missing to separate the two new lists that you've added

Reply:
1

Reply:
Thanks for alerting me to this, I've added the missing commas